tests/driver/napi/nodejs.rs

```diff
@@ -0,0 +1,80 @@
+// Copyright © SixtyFPS GmbH <info@slint.dev>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-Royalty-free-1.1 OR LicenseRef-Slint-commercial
+
+use std::error::Error;
+use std::{fs::File, io::Write, path::PathBuf};
+
+lazy_static::lazy_static! {
+    static ref NODE_API_JS_PATH: PathBuf = {
+        let  node_dir = PathBuf::from(env!("CARGO_MANIFEST_DIR")).join("../../../api/napi");
+
+        // On Windows npm is 'npm.cmd', which Rust's process::Command doesn't look for as extension, because
+         // it tries to emulate CreateProcess.
+         let npm = which::which("npm").unwrap();
+
+        // Ensure TypeScript is installed
```

Comment:
This comment seems wrong, as npm install for napi's package.json does more.

Reply:
I have updated the comment.